app/models/ability.rb

```diff
@@ -9,6 +9,8 @@ def initialize(user)
         can :manage, :all
       else
         can :manage, :all
```

Comment:
Applicants should also probably be able to manage submissions (create them to make an "application")...